client/js/components/statement/fragment/SelectDocument.vue

```diff
@@ -93,79 +93,79 @@ export default {
   name: 'DpSelectDocument',
 
   props: {
-    elements: {
+    documents: {
       required: true,
-      type: Array,
-      default: () => []
-    },
-
-    paragraphs: {
       type: Object,
-      required: true,
       default: () => ({})
     },
 
-    documents: {
-      type: Object,
+    elements: {
       required: true,
-      default: () => ({})
+      type: Array,
+      default: () => []
```

Comment:
I know you didn't introduce it, so just as a side note: If a prop is required, a default is not needed (as it will never be used)